projects/ccf-rui/src/app/core/store/model/model.state.ts

```diff
@@ -167,13 +167,13 @@ export class ModelState extends NgxsImmutableDataRepository<ModelStateModel> {
     if (this.globalConfig.organ) {
       const organConfig = this.globalConfig.organ;
       const organName = organConfig.name.toLowerCase();
-      const organInfo = ALL_ORGANS.find((o) => o.name.toLowerCase() === organName);
+      const organInfo = ALL_ORGANS.find((o) => o.organ.toLowerCase() === organName);
```

Comment:
Need to mach organ and side.. From ingest portal, they will provide organ name "kidney" and organ side "left"